api/sessions.py

```diff
@@ -115,11 +115,19 @@ def get(self, pid=None, gid=None):
             query = {'group': gid}
         else:
             query = {}
-        projection = ['label', 'subject_code', 'subject.code', 'project', 'group']
+        projection = ['label', 'subject_code', 'subject.code', 'project', 'group', 'subject.age', 'subject.sex']
         sessions = self._get(query, projection, self.request.GET.get('admin', '').lower() in ('1', 'true'))
+        session_measurements = {}
+        if self.request.GET.get('measurements', '').lower() in ('1', 'true'):
+            session_measurements = self.app.db.acquisitions.aggregate([
+                {'$match': {'session': {'$in': [sess['_id'] for sess in sessions]}}},
+                {'$group': {'_id': '$session', 'measurements': {'$addToSet': '$datatype'}}}
+                ])
+            session_measurements = {sess['_id']: sess['measurements'] for sess in session_measurements}
         for sess in sessions:
+            sess['measurements'] = session_measurements.get(sess['_id'], None)
```

Comment:
To me, it looks like this line will overwrite any pre-existing `measurements` key on the object if `measurements` was not passed as a URL param. Is there any case where that might be a problem?